hera_cal/abscal.py

```diff
@@ -118,17 +117,22 @@ def __init__(self, model, data, wgts=None, antpos=None, freqs=None, pol_select=N
         pol_select : list of polarizations you want to keep in data
                      type=list, dtype=str, Ex. ['xx', 'yy']
         """
+        # set pols to None
         pols = None
+
         # check format of model
         if type(model) == list or type(model) == np.ndarray or type(model) == str or type(model) == UVData:
-            (model, model_flags, model_antpos, model_ants, model_freqs,
+            (model, model_flags, model_antpos, model_ants, model_freqs, model_lsts,
              model_times, model_pols) = UVData2AbsCalDict(model, pop_autos=True, return_meta=True, pol_select=pol_select)
 
         # check format of data
```

Comment:
Clarify in comments that this is for when the model is not a dictionary already.